### PR TITLE
WIP: Code style and general improvements

### PR DESCRIPTION
**Pull request by colinodell**
See #9 

Remaining tasks:

- [x] Rebase onto latest `master`
- [x] Make StyleCI happy
- [x] Add `LICENSE`
- [x] Add `CHANGELOG`
- [x] Add branch alias in `composer.json`
- [x] Other stuff?

### CODE REVIEW

File: src/Alexa.php [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<?php namespace Develpr\AlexaApp;
```
Comment (develpr) [inline src/Alexa.php]: Thanks for this. I don't remember WHEN, but a while back I believe this was something of a "best practice" that somebody put forth. Maybe it wasn't *actually* a best practice, but somewhere I read something that stuck in my head that this was the right way to do things. So it's been a habit of sorts. I just went through a few random Laravel and Symfony classes and sure enough, they don't do it this way! Thanks for triggering me to look :)

Comment (colinodell) [inline src/Alexa.php]: Sure thing!

**Comment by colinodell** [comment]
@develpr I think I'm done :)

I'm using this branch in my Laravel application which has two custom intents - it seems to work perfectly.  However, if you'd like to do additional testing I'd absolutely welcome it!  And let me know if there are any additional changes you'd like to see.

File: composer.json [hunk 2]
```diff
@@ -19,6 +19,11 @@
     "autoload": {
         "psr-4": {
             "Develpr\\AlexaApp\\": "src"
-           }
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "0.4-dev"
```
Comment (colinodell) [inline composer.json]: @develpr When you release a new `0.x.0` version, bump this to the next-highest one.

For example, just before you release `0.4.0`, change this value to `"0.5-dev"`.  That'll tell Composer to consider `master` as the development branch of `0.5.0`.  (You can keep releasing `0.4.x` versions while this is set to `0.5-dev` - that's totally fine.)

**Pull request merged**